is.js

```diff
@@ -768,12 +768,40 @@
         if(is.not.array(arr)) {
             return false;
         }
-        for(var i = 0; i < arr.length; i++) {
-            if(arr[i] > arr[i + 1]) return false;
-        }
-        return true;
+
+        return checkArrayAsc(arr) || checkArrayDesc(arr);
     };
 
+    function checkArrayAsc(arr) {
+      for(var i = 0; i < arr.length; i++) {
```

Comment:
we can loop only till `arr.length - 1` instead of `arr.length` here and avoid the `undefined` check for next below.